net/netatalk/Portfile

```diff
@@ -1,81 +1,68 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           openssl 1.0
+PortGroup           github 1.0
+PortGroup           meson 1.0
 
 name                netatalk
-version             2.2.10
+subport             netatalk4 {}
+
+# Set up mutual conflicts between subports
+if {${subport} eq "netatalk4"} {
+    conflicts       netatalk
```

Comment:
`netatalk4` requires C11, that should be specified, otherwise a wrong compiler can be picked, and the build fails.

Reply:
The code you have slected here is not relevent to your comment around c11. If you can link me to examples or docs I can have  a look.